book/opmanual_duckiebot/atoms_30_demos/16_unsupervised_image_segmentation.md

```diff
@@ -54,14 +48,14 @@ camera_kmean.launch file is located under /catkin_ws/src/05-teleop/pi_camera/lau
     laptop $ dts start_gui_tools ![ROBOT_NAME]
     root@user $ rqt_image_view
```

Comment:
laptop-container ?

Reply:
We have to type `dts start_gui_tools ![ROBOT_NAME]` on the laptop it takes us to the root of the laptop and there we open rqt window using rqt_image_view

Reply:
Same here you can write "laptop-container $"